source/mesh_deformation/fastscape.cc

```diff
@@ -1698,6 +1736,11 @@ namespace aspect
           prm.declare_entry("Initial noise magnitude", "5",
                             Patterns::Double(),
                             "Maximum topography change from the initial noise. Units: $\\{m}$");
+          prm.declare_entry("Additional output", "Bedrock river incision rate",
+                            Patterns::Selection("Bedrock river incision rate|Bedrock deposition coefficient|Uplift rate"),
```

Comment:
Whereas the names of parameters start with upper-case letters, the *values* are almost always lower-case letters. So let's choose this:
```suggestion
                            Patterns::Selection("bedrock river incision rate|bedrock deposition coefficient|uplift rate"),
```